qemu/tests/cfg/block_check_max_tranfer_length.cfg

```diff
@@ -0,0 +1,19 @@
+- block_check_max_tranfer_length:
+    type = block_check_max_tranfer_length
+    only virtio_blk
+    only Windows
+    only x86_64, i386
+    images += " stg"
+    image_size_stg = 10G
+    image_name_stg = images/stg
+    create_image_stg = yes
+    remove_image_stg = yes
+    force_create_image_stg = yes
+    check_cmd = "WIN_UTILS:\spti.exe %s:"
+    variants:
+         - with_queue_size_16:
```

Comment:
```suggestion
        - with_queue_size_16:
Suggest keeping 4 blank spaces.

Reply:
@YongxueHong  Many thanks, It is really a good checkpoint. I will pay attention to it next time.

```
 (1/2) Host_RHEL.m8.u5.product_av.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_check_max_tranfer_length.with_queue_size_16.q35: STARTED
 (1/2) Host_RHEL.m8.u5.product_av.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_check_max_tranfer_length.with_queue_size_16.q35: PASS (139.16 s)
 (2/2) Host_RHEL.m8.u5.product_av.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_check_max_tranfer_length.with_queue_size_32.q35: STARTED
 (2/2) Host_RHEL.m8.u5.product_av.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_check_max_tranfer_length.with_queue_size_32.q35: PASS (93.57 s)
```
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0